### ISSUE TITLE
Wifi Disconnect after period of inactivity

### ISSUE THREAD

**Issue by MakerDockio** [austinscreations/OXRS-AC-TP32-ESP-FW#38]
I've been getting this issue since the first version of the firmware however after loading the latest firmware this issue still seems to persist for me. 

If I leave the Touch Panel for a day or two without interacting with it I get the ⚠️ in the bottom right-hand corner of the footer, if I go to settings I see there isn't an IP address. I have monitored my network during the last 4 occasions and the network is rock solid. 

The device remains with the  ⚠️ in the bottom right-hand corner of the footer, and never seems to reattempt a connection to get an IP. To resolve this every few days I have to power down/ up the device and it auto reconnects. 

This happens consistently if the Touch Panel is left for a period of 1 / 2 days, given that this period of inactivity is quite likely for a household with multiple of these devices it seems this may need to be looked into further.

**Comment by plains203** [comment]
The screen in my office regularly sits that long without being touched over the weekend. It's not currently on the latest firmware but I haven't seen that issue. I did see a random reboot on it and my Screen at one point but I haven't seen that for weeks. I'll update my office screen and see if it drops out or not. My Screens are set to static ip addresses so I will set one up with dhcp and see how it goes. Out of interest how long are you dhcp leases set to on your dhcp server?

**Comment by MakerDockio** [comment]
Good point and sorry for not stating it before, I also have all my devices set with static IP's. I also have multiple AP's and have the devices locked to specific AP's for maximum wifi performance.

It is worth mentioning this did also occur as frequently before I had static IP and locked devices to AP's I made this change whilst investigating potential issues on my network side. However it seems that it hasn't made a difference.

**Comment by plains203** [comment]
I can confirm that with my unifi setup, if I kick a screen from the wifi it doesn't appear to attempt to reconnect once it is unblocked via the unifi admin portal.

**Comment by plains203** [comment]
https://github.com/tzapu/WiFiManager/issues/1454 this may have the answer in it.

**Comment by sumnerboy12** [comment]
Nice find @plains203 - that looks very likely to be the same problem as #3. Not sure it is related to this issue that @MakerDockio is seeing?

**Comment by plains203** [comment]
After reading both issues it would appear they are related. Potentially if you updated the other devices you mentioned in the other issue you would be able to make them fail as well.
